ui/apps/platform/src/Containers/AccessControl/PermissionSets/ResourceDescription.tsx

```diff
@@ -13,6 +13,7 @@ const resourceDescriptions: Record<ResourceName, string> = {
         'Read: N/A Write: Delete comments from other users. All users can edit and delete their own comments by default. To add and remove comments or tags, you need a role with write access for the resource you are modifying.',
     CVE: 'Internal use only',
     Cluster: 'Read: View secured clusters. Write: Add, modify, or delete secured clusters.',
+    ClusterCVE: 'Internal use only',
```

Comment:
Do we have to add it within the UI as a new resource? I know we have other resources that are being leaked to the UI that are for internal use only, but why add another one?

Reply:
`ClusterCVE` is returned as a resource from the backend, this change just makes sure it's description is not empty

Reply:
I see. It's unfortunate that we return it then in the first place, I suppose such is life then for the internal resources.